genisys/main.py

```diff
@@ -66,3 +66,32 @@ def main():
 
 if __name__ == "__main__":
     main()
+
+'''
+import needed modules,
+create new module with options/configs from other branches.
+'''
+def run():
+
+    #netplan.py
+    import modules.netplan as net
+
+    #get settings for functions
+    netplan = net.Netplan()
+    netLocation = net.install_location()
+    netgenerate = net.generate()
```

Comment:
Generate should not have to be called manually on modules. Lets talk about the class inheritance architecture on the modules at some point.